src/include/codegen/operator/table_scan_translator.h

```diff
@@ -17,6 +17,8 @@
 #include "codegen/operator/operator_translator.h"
 #include "codegen/scan_callback.h"
 #include "codegen/table.h"
+#include "type/value.h"
+#include "codegen/proxy/runtime_functions_proxy.h"
```

Comment:
Doesn't look like these are needed in the header file. Either remove entirely or move to cpp if they're needed.